packages/wallet/Account.js

```diff
@@ -0,0 +1,604 @@
+const rp = require('request-promise');
+const StellarSdk = require('stellar-sdk');
+const Web3 = require('web3');
+const openurl = require('openurl');
+const toml = require('toml');
+
+const server = new StellarSdk.Server('https://horizon-testnet.stellar.org');
+const web3 = new Web3('https://rinkeby.infura.io/v3/54add33f289d4856968099c7dff630a7');
+
+/**
+ * @class Account
+ * This is a wrapper class over stellar and ethereum accounts.
+ * For those fields/methods that are already there when the original
+ * account is passed in, this class simply extracts them;
+ * Otherwise, the fields/methods are created and saved in this class.
+ */
+class Account {
+  /**
+  * constructor
+  * @param {string} network
+  */
+  constructor(network) {
+    switch (network) {
+      case 'stellar':
+        this.network = 'stellar';
+        this.testAddress = 'https://horizon-testnet.stellar.org/accounts/';
+        StellarSdk.Network.useTestNetwork();
+        break;
+      case 'ethereum':
+        this.network = 'ethereum';
```

Comment:
I think it is good to list out all the fields that this class can have e.g `this.balance`, etc for a clearer  "init" view